cves/kernel/CVE-2021-27365.yml

```diff
@@ -207,10 +212,12 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+   This bug was found by Adam Nichols, who is not a RedHat employee. He said
+   he was reading through a subsystem in the linux kernel source code, and 
```

Comment:
I think you forgot to finish this sentence

Reply:
Fixed, thank you!